src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java

```diff
@@ -58,14 +58,13 @@ public void prepareTest() {
   }
 
   //TODO: Move logging methods to page object

```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)